inversion-api/src/main/java/io/inversion/cloud/utils/RestClient.java

```diff
@@ -78,35 +82,42 @@
  */
 public class RestClient
 {
-   static Log                                       log             = LogFactory.getLog(RestClient.class);
+   static Log                                       log              = LogFactory.getLog(RestClient.class);
 
    //-- config properties
-   protected String                                 name            = null;
-   protected String                                 url             = null;
-   protected boolean                                remoteAsync     = true;
-   protected boolean                                localAsync      = false;
-   protected ArrayListValuedHashMap<String, String> forcedHeaders   = new ArrayListValuedHashMap();
-   protected boolean                                forwardHeaders  = false;
-   protected boolean                                forwardParams   = false;
+   protected String                                 name             = null;
+   protected String                                 url              = null;
+   protected boolean                                remoteAsync      = true;
+   protected boolean                                localAsync       = false;
+   protected ArrayListValuedHashMap<String, String> forcedHeaders    = new ArrayListValuedHashMap();
+
+   //https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers
+   protected boolean                                forwardHeaders   = false;
+   protected Set                                    whitelistHeaders = new HashSet(Arrays.asList("authorization", "cookie", "x-forwarded-host", " x-forwarded-proto"));
+   protected Set                                    blacklistHeaders = new HashSet();
```

Comment:
I think we should go ahead and definitely add the following headers to the blacklist:

content-length
content-type
content-md5

As each request will want its own values set for these.